common/interactive-object/server.mjs

```diff
@@ -259,6 +262,52 @@ class InteractiveObjectServerBaseClass {
 
     return this.sf_exit(playerID, {next: this.dataStore.get(playerID)});
   }
+
+  /**
+   * Check permission by JWT token and go to corresponding state
+   * @param {String} playerID
+   * @param {Object} kwargs - TODO
+   * @return {String} - the next state
+   */
+  async sf_checkPermission(playerID, kwargs) {
+    const permission = await this.helper.getToken(playerID);
+    permission.scope.push('online_bulletin_editor');
```

Comment:
Did you leave this here for testing?

Reply:
Yes. Because there is no scope called "online_bulletin_editor" now, I will add a TODO to remember to delete it later.